src/opts.rs

```diff
@@ -72,7 +72,7 @@ pub enum Subcommand {
         output_dir: Option<PathBuf>,
     },
     /// List contents of an archive
-    #[command(visible_alias = "l")]
+    #[command(visible_alias = "l", visible_alias = "ls")]
```

Comment:
to me this is a little more readable (and shorter)

```suggestion
    #[command(visible_aliases = ["l", "ls"])]
```

Reply:
D'oh, you're right!

Reply:
PR is updated.